### DISCUSSION TITLE
Altimeter Value?

### DISCUSSION THREAD

**Discussion by NIH301** [nguyenquyhy/Flight-Tracker-StreamDeck#304] [Q&A] [answered]
Hi There, 
I want to display the current Altimeter Value. I have found the way to increase/decrease the value, but can't find any way to display current value. Any help appreciated.

**Comment by rmroc451** [comment, marked as answer]
https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/265eca504044c6158856965c044a3ddb68a804ae/FlightStreamDeck.Core/TOGGLE_VALUE.cs#L211

**Reply by NIH301** [reply]
Thank you so much! I would never have found that myself!
